gradle/component.gradle

```diff
@@ -144,6 +142,7 @@ kotlin {
                 useExperimentalAnnotation 'kotlinx.coroutines.InternalCoroutinesApi'
                 useExperimentalAnnotation 'kotlinx.coroutines.FlowPreview'
                 useExperimentalAnnotation 'kotlin.ExperimentalUnsignedTypes'
+                useExperimentalAnnotation 'kotlin.ExperimentalStdlibApi'
```

Comment:
it would be nice to also use this syntax for example rather than setting a flag.

Reply:
I tried, but it didn't work. Will check one more time

Reply:
Ok, not mandatory.

Reply:
Managed to resolve